packages/video_player/video_player/example/lib/main.dart

```diff
@@ -250,17 +255,95 @@ class _BumbleBeeRemoteVideoState extends State<_BumbleBeeRemoteVideo> {
         children: <Widget>[
           Container(padding: const EdgeInsets.only(top: 20.0)),
           const Text('With remote mp4'),
+          FutureBuilder<bool>(
+            key: _pictureInPictureKey,
+            future: _controller.isPictureInPictureSupported(),
```

Comment:
Doesn't most of the UI need to be conditional on this? It looks like the button below to trigger PiP is unconditionally enabled and will call a method that isn't implemented on most platforms, for instance.